src/chocolatey.resources/helpers/functions/Get-WebFile.ps1

```diff
@@ -45,10 +60,10 @@ param(
       $creds = $cred.GetNetworkCredential();
     }
     $proxyaddress = $webclient.Proxy.GetProxy($url).Authority
-    Write-host "Using this proxyserver: $proxyaddress"
+    Write-Host "Using system proxy server '$proxyaddress'."
     $proxy = New-Object System.Net.WebProxy($proxyaddress)
-    $proxy.credentials = $creds
-    $req.proxy = $proxy
+    $proxy.Credentials = $creds
+    $req.Proxy = $proxy
   }
 
   $req.Accept = "*/*"

```

Comment:
Even better, the Get-WebHeaders.ps1 file is considered a binary, likely due to encodings. More :/